### ISSUE TITLE
Merging should update the data size column

### ISSUE THREAD

**Issue by dkull** [postgrespro/pg_probackup#40]
After merging DELTAS to FULL the data size column shows the table is 16MB in size but should be ~2GB. I haven't tried restoring from it yet, but the backup dir is still ~2GB in size. (It is in /tmp/ dir intentionally, I am testing out probackup). Also it seems that Start LSN shows fonly last DELTA data. Did pg_probackup lose data?

`pg_probackup-10 2.0.25 (PostgreSQL 10.6)`

```
tanel@koljat:/tmp$ sudo pg_probackup-10 show -B /tmp/pgbackup/ --instance mypg10
====================================================================================================================================
 Instance  Version  ID      Recovery Time           Mode   WAL     Current/Parent TLI  Time    Data   Start LSN    Stop LSN  Status
====================================================================================================================================
 mypg10    10       PKNM58  2019-01-01 15:15:22+02  DELTA  STREAM  1 / 0                20s    16MB  1/E9000028  1/E9000198  OK
 mypg10    10       PKMT4I  2019-01-01 04:48:32+02  DELTA  STREAM  1 / 0                21s    17MB  1/E7000028  1/E7000198  OK
 mypg10    10       PKMNHY  2019-01-01 02:47:08+02  FULL   STREAM  1 / 0                26s  2087MB  1/E5000028  1/E5000198  OK
tanel@koljat:/tmp$ sudo pg_probackup-10 merge -B /tmp/pgbackup/ --instance mypg10 -i PKNM58
INFO: Merge started
INFO: Merging backup PKMT4I with backup PKMNHY
INFO: Validating backup PKMNHY
INFO: Backup PKMNHY data files are valid
INFO: Validating backup PKMT4I
INFO: Backup PKMT4I data files are valid
INFO: Delete: PKMT4I 2019-01-01 04:48:32+02
INFO: Rename PKMNHY to PKMT4I
INFO: Merging backup PKNM58 with backup PKMT4I
INFO: Validating backup PKNM58
INFO: Backup PKNM58 data files are valid
INFO: Delete: PKNM58 2019-01-01 15:15:22+02
INFO: Rename PKMT4I to PKNM58
INFO: Validating backup PKNM58
INFO: Backup PKNM58 data files are valid
INFO: Merge of backup PKNM58 completed
tanel@koljat:/tmp$ sudo pg_probackup-10 show -B /tmp/pgbackup/ --instance mypg10
=================================================================================================================================
 Instance  Version  ID      Recovery Time           Mode  WAL     Current/Parent TLI  Time  Data   Start LSN    Stop LSN  Status
=================================================================================================================================
 mypg10    10       PKNM58  2019-01-01 15:15:22+02  FULL  STREAM  1 / 0                20s  16MB  1/E9000028  1/E9000198  OK
tanel@koljat:/tmp$ sudo du -sh /tmp/pgbackup/
2,0G    /tmp/pgbackup/

```

**Comment by gsmol** [comment]
Thank you for the feedback!
I think it was already fixed in 34c05740b05dd1 and soon will be shipped with 2.0.26.

**Comment by dkull** [comment]
Thank you! I tried to find a duplicate but it seems I missed it.
